xmodule/library_content_block.py

```diff
@@ -90,19 +91,24 @@ class LegacyLibraryContentBlock(ItemBankMixin, XModuleToXBlockMixin, XBlock):
         display_name=_("Library Version"),
         scope=Scope.settings,
     )
-    max_count = Integer(
-        display_name=_("Count"),
-        help=_("Enter the number of components to display to each student. Set it to -1 to display all components."),
-        default=1,
-        scope=Scope.settings,
-    )
     capa_type = String(
         display_name=_("Problem Type"),
         help=_('Choose a problem type to fetch from the library. If "Any Type" is selected no filtering is applied.'),
         default=ANY_CAPA_TYPE_VALUE,
         values=_get_capa_types(),
         scope=Scope.settings,
     )
+    # This is a hidden field that stores whether child blocks are migrated to v2, i.e., whether they have an upstream.
+    # We cannot completely remove this block code until we force-migrate course content. Otherwise, we'll lose student
+    # data (such as selected fields), which tracks the children selected for each user.
```

Comment:
We will never force-migrate course content

```suggestion
    # We can never completely remove the legacy library_content block; otherwise, we'd lose student
    # data, such as selected fields, which tracks the children selected for each user.
```